consensus/polybft/extra.go

```diff
@@ -459,6 +460,13 @@ func (c *CheckpointData) Validate(parentCheckpoint *CheckpointData,
 		return fmt.Errorf("epoch number should not change for epoch-ending block")
 	}
 
+	// exit root hash of proposer and
+	// validator that validates proposal have to match
+	if exitRootHash != c.EventRoot {
+		return fmt.Errorf("exit root hash not as expected. Expected: %s. Actual: %s",
```

Comment:
Nitpick: not sure if we should log expected and actual event roots. Could they be exploited somehow?
At least that was my mindset when I was validating current and next validators' hashes. I wasn't confident enough to log it due to some security considerations.

Reply:
I don't think so, because they are `keccak256` hashes. I just put it there to see if they really don't match, but I can remove the log.

Reply:
Then IMO either remove the hash from the error message here or expand the error message when we are checking validator set hashes to make error messages consistent.

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1638/commits/ae88b4fee5735c76092d96575430705e62bcba9a